hack/build-acceptable-bundles.sh

```diff
@@ -3,10 +3,27 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+
+# Function to remove the sha and digest from the image name
+# from: quay.io/konflux-ci/task1:0.1-1234@sha256:5678 to quay.io/konflux-ci/task1:0.1
+strip_image_tag() {
+  sed 's/\(:[^-]*\).*/\1/' <<< "$1"
+}
+
 # helps with debugging
 DATA_BUNDLE_REPO="${DATA_BUNDLE_REPO:-quay.io/konflux-ci/tekton-catalog/data-acceptable-bundles}"
 mapfile -t BUNDLES < <(cat "$@")
 
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+# File containing the list of images
+OUTPUT_TASK_BUNDLE_LIST="${OUTPUT_TASK_BUNDLE_LIST-${SCRIPTDIR}/../task-bundle-list}"
+for i in "${!BUNDLES[@]}"; do
+    original_line="${BUNDLES[$i]}"
+    modified_line=$(strip_image_tag "$original_line")
```

Comment:
I just realized that we pull the image in `ec track bundle`, so the stripping of the task hash from the tag will break `ec track bundle`, see https://github.com/enterprise-contract/ec-cli/blob/5cf4a7ff5d29729ce42150b030f2f1f2b4140b5d/internal/tracker/tracker.go#L160 and https://github.com/enterprise-contract/ec-cli/blob/5cf4a7ff5d29729ce42150b030f2f1f2b4140b5d/internal/tracker/bundle_info.go#L30.

That is the image reference here needs to be pushed already with the release tag for `ec track bundle` to work.

Reply:
Given that the digests are present in the image references, this will work even without the images being tagged with the release tag (e.g. `0.X`).